readinglist/__init__.py

```diff
@@ -72,9 +72,9 @@ def on_new_request(event):
     config.add_subscriber(on_new_request, NewRequest)
 
     def on_new_response(event):
-        # Add request revision info in response headers.
-        revision = six.text_type(getattr(event.request, 'revision', ''))
-        event.request.response.headers['Timestamp'] = revision.encode('utf-8')
+        # Add request timestamp info in response headers.
+        timestamp = six.text_type(getattr(event.request, 'timestamp', ''))
```

Comment:
I don't want to return an empty header in case we don't have any timestamp, we should return the server timestamp.
Also I wonder if we don't want to return the real server Timestamp in Timestamp and the Last-Modified header with the current collection timestamp.

Last point is that I think this db.timestamp thing should be related to the user collection and not only the user, as soon as we will have to manage more than one collection we will have problems.